index.bs

```diff
@@ -1127,6 +1127,12 @@ BCP 14 [[!RFC2119]] [[!RFC8174]] when, and only when, they appear in all capital
             when the [=public key credential source=] was [=registration|registered=].
             Storing this in combination with the above [$credential record/attestationObject$] [=struct/item=]
             enables the [=[RP]=] to re-verify the [=attestation signature=] at a later time.
+        
+        :   <dfn>rpId</dfn>
+        ::  The value of the <code>{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> parameter
+            specified in the {{CredentialsContainer/create()}} operation during credential registration.
+            Storing this enables the [=[RP]=] to use the credential across different domains later
```

Comment:
```suggestion
            This is a core property of a credential and storing the value at creation time is recommended to assist with future operations such as audits, troubleshooting, or leveraging features such as  [[#sctn-related-origins|Related Origin Requests]].
```

Reply:
some starter text. needs some work.

Reply:
Thanks for the inspiration, I made some changes:

> **rpId**
> The value of the rp.id parameter specified in the create() operation during credential registration. This value is a core property of the credential that determines where it can be used. Storing this value at registration time can help in the future, such as to audit its use, troubleshoot issues authenticating with it, or to use it across different domains later via Related Origins.